app/Decsys/Repositories/LiteDb/LiteDbWebhookRepository.cs

```diff
@@ -9,7 +10,7 @@ namespace Decsys.Repositories.LiteDb;
 /// </summary>
 public class LiteDbWebhookRepository : IWebhookRepository
 {
-    public int Create(WebhookModel webhook)
+    public ObjectId Create(WebhookModel webhook)
```

Comment:
LiteDb can't use Mongo stuff - change this to `string` as per the updated interface